src/com/epam/javatrainings/multiThread/Main.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.javatrainings.multiThread;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int minute = scanner.nextInt();
+        int second = scanner.nextInt();
```

Comment:
Should present some beginning message like(please enter preferable minute/second)

Reply:
okay )